README.md

```diff
@@ -4,12 +4,12 @@
 
 ```yaml
 jobs:
-  build:
+  update_external_airflow_fork:
     runs-on: ubuntu-latest
     steps:
       uses: TobKed/github-forks-sync-action@master
       with:
-        github_token: ${{ secrets.GH_TOKEN }}
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@turbaszek now I figured out that `secrets.GITHUB_TOKEN` is not a valid one. As the [docs](https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#about-the-github_token-secret) says about `GITHUB_TOKEN`:

> The token's permissions are limited to the repository that contains your workflow.

This example is for updating external repository and requires Personal Access Token (https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)

Reply:
It would be good to mention what privilages should have the secret